movement/io/load_poses.py

```diff
@@ -370,6 +380,15 @@ def from_dlc_file(
     >>> from movement.io import load_poses
     >>> ds = load_poses.from_dlc_file("path/to/file.h5", fps=30)
 
+    Notes
+    -----
+    In movement, pose data can only be loaded if all individuals have
+    the same set of keypoints (i.e., the same labeled body parts).
+    While DeepLabCut supports assigning different keypoints to different
+    individuals (see the `DeepLabCut documentation for multi-animal project,
+    <https://deeplabcut.github.io/DeepLabCut/docs/maDLC_UserGuide.html#b-configure-the-project>`_),
+    this feature is not currently supported in movement.
```

Comment:
```suggestion
    In ``movement``, pose data can only be loaded if all individuals have
    the same set of keypoints (i.e., the same labeled body parts).
    While DeepLabCut supports assigning keypoints that are not shared across
    individuals (see the `DeepLabCut documentation for multi-animal projects
    <https://deeplabcut.github.io/DeepLabCut/docs/maDLC_UserGuide.html#b-configure-the-project>`_),
    this feature is not currently supported in ``movement``.
```

Same changes as in the Input/Output guide, but note that here we need two backticks to achieve monospace formatting, because docstrings are in .rst (restructured text) format as opposed to markdown.